src/routes/docs/tutorials/nuxt/step-4/+page.markdoc

```diff
@@ -0,0 +1,175 @@
+---
+layout: tutorial
+title: Add authentication
+description: Add authentication to your Nuxt application using Appwrite Web SDK
+step: 4
+---
+
+For our ideas tracker app, we want any visitor to be able to read the ideas that are stored.
+On the other hand, we don't want the page spammed with just about anything from anyone just stopping by.
+To prevent that, or at least making it a bit more difficult, editing ideas will be available for logged in users only.
+With a login function, we can differentiate between users and decide which users have access to which content.
+
+We will build a page with a simple login form and store its related logic in a `useUserSession` composable so it can be reused, starting with the composable.
+
+# User session composable {% #user-session-composable %}
+
+There are a few standard functions involved in handling a user session that is added to the composable.
+The user needs to be able to register to an account, login to the account and logout from it.
+
+We are using Appwrite as a backend to handle the user details, so we need to connect to Appwrite by importing the configurations from step 3.
+The response from these interactions will be stored as references to get more information about the user in our app.
+
+In your `composable` directory, create the file `useUserSession.js` and add the following code.
+Then you can call the  `useUserSession()` function in the pages and components to use the functionality.
+
+```js
+// composable/useUserSession.js
+
+import { ID } from "appwrite";
+import { ref } from "vue";
+import { account } from "/appwrite";
+
+const current = ref(null) //Reference to current user object
```

Comment:
I also worry this may be dangerous due to SSR. In the server, this can be shared with other users. I don't recall how Nuxt works in this sense, but in general this is something to be wary of.

Reply:
Alternatively, disable SSR in the app. IIRC it's what I did for the Svelte tutorial, as much as it pains me. We can update it once we have SDK SSR support.

Reply:
Fixed